test/integration/backward_compatible/parallel/serialization/compact/CompactUtil.js

```diff
@@ -127,6 +135,29 @@ if (TestUtil.isClientVersionAtLeast('5.1.0')) {
         }
     };
 
+    DefaultObjectSampleClass = class {
+        constructor(id) {
+            this.id = id;
+        }
+    };
+    DefaultObjectSampleClassSerializer = class DefaultObjectSampleClassSerializer {
```

Comment:
What about DefaultSerializerOverridingSerializer ?

Reply:
Fixed.

Reply:
Now that we removed the class of this serializer, we can change typename from "DefaultObjectSampleClass" to something else, a random string should be fine. Like this:

```js
getTypeName() {
    return TestUtil.randomString();
}
```

Reply:
Fixed